plugins/woocommerce-blocks/packages/components/text-input/validated-text-input.tsx

```diff
@@ -232,10 +232,6 @@ const ValidatedTextInput = forwardRef<
 		}
 
 		const hasError = validationError?.message && ! validationError?.hidden;
-		const describedBy =
```

Comment:
We don't want to overwrite the `aria-describedby` attribute when there is an error - the `aria-describedby` should not change when an error is present.